pandas/core/reshape/reshape.py

```diff
@@ -940,6 +941,7 @@ def wide_to_long(df, stubnames, i, j, sep="", suffix='\d+'):
     8      3      3  2.1  2.9
     >>> l = pd.wide_to_long(df, stubnames='ht', i=['famid', 'birth'], j='age')
     >>> l
+    ... # doctest: +NORMALIZE_WHITESPACE
```

Comment:
@jorisvandenbossche I think the issue was when there's a `df.index.name` the output has whitespace for the rest of that line  (which we don't want to include in the source)